playwright/tests/test_17_EstTool_UI_AssumpLimitations.py

```diff
@@ -96,7 +98,6 @@ def test_example(page: Page) -> None:
     expect(page.get_by_text("These figures are combined")).to_be_visible()
     expect(page.get_by_role("heading", name="Network Data Transfer")).to_be_visible()
     expect(page.get_by_text("Our outgoing network data")).to_be_visible()
-    expect(page.get_by_text("Assumptions and Limitations The Technology Carbon Estimator tool is designed to")).to_be_visible()
```

Comment:
This was causing the test to fail, I think because there is now an 'X' button in between the first and second halves of the test. I removed this expectation because I think it's a duplicate of some of the expectations [above](https://github.com/ScottLogic/sl-tech-carbon-estimator/pull/96/files#diff-0574963fc62f5929ba3c969da81828750d7d54fbcf5b883e54416d6c983313e2R64).